### ISSUE TITLE
Request for support on xds

### ISSUE THREAD

**Issue by hzxuzhonghu** [spring-cloud/spring-cloud-kubernetes#925]
We have met many users who want to use istio, in order to do this users need to drop many features provided by spring cloud. 

I have an idea, could springcloud support xds, so it could connect to any management server and communicate with it via xds. It is beneficial for both spring cloud and the popular mesh concept. BTW, grpc has recently supported xds

**Comment by ryanjbaxter** [comment]
Are you imagining a `DiscoveryClient` xDS implementation?

**Comment by hzxuzhonghu** [comment]
Yes, definitely need a discovery client and after it gets xds from server, it need to convert the config to what springcloud knows

**Comment by ryanjbaxter** [comment]
If this is something we see demand for we can certainly consider it

**Comment by hzxuzhonghu** [comment]
Thanks. We want to make our spring cloud app communicate with service mesh ones, and behave like those.

**Comment by fnarvaez-applydigital** [comment]
Hello :) any news about this?
